### PR TITLE
Improve the durgot k3x0 readme.md

### PR DESCRIPTION
**Pull request by ogabriel**
## Description

First of all, I'm a beginner when it comes to QMK, and just started using it this week. And glad to the development that went in this project in all fronts!

But when trying to flash my Durgod K320, I had some problems understanding the current docs, they are definitely not bad, but as a beginner, some of the terms or actions described on the original readme didn't really make sense at first glance, and I was only able to understand after searching on the discord server for previous questions like mine or researching on the internet.

So after flashing my keyboard for the first time, and getting more knowledgeable on the matter, I've writter a more descriptive readme for this keyboard, including requirements and a more descriptive behavior of the keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/durgod/k3x0/readme.md [hunk 1]
```diff
@@ -18,34 +18,58 @@ Instructions for building the K310 and K320 firmware can be found here:
...
-You can short Boot0 to Vdd by shorting R21 to C27 on the sides closest to the processor, as shown:
+1. Be in Bootloader mode.
+2. Here are a few options for performing the initial Flash:
+  - Using [QMK Toolbox](https://github.com/qmk/qmk_toolbox)
+  - Using dfu-util:
 
-<img src="https://i.imgur.com/hvDnw5a.jpg" width="520" height="693" alt="Shorting Boot0 to Vdd on K320">
+```bash
+dfu-util --alt 0 --dfuse-address 0x08000000 --download ${NEW_QMK_BIN}
+```
```
Comment (fauxpark) [inline keyboards/durgod/k3x0/readme.md:67]: `qmk flash` as normal should be fine here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/durgod/k3x0/readme.md (hunk 1 above)
Comment (ogabriel) [reply keyboards/durgod/k3x0/readme.md:67]: i tested and really worked, i will update it

File: keyboards/durgod/k3x0/readme.md [hunk 2]
```diff
@@ -18,34 +18,58 @@ Instructions for building the K310 and K320 firmware can be found here:
...
+
+If it does not behave like described, remove the cable and go back to step 4.
+
+#### Backup original firmware
+
+1. Be in Bootloader mode.
+2. Execute this command to backup:
   - Using dfu-util:
+```bash
+dfu-util --alt 0 --dfuse-address 0x08000000 --upload ${OLD_STOCK_BIN}
```
[the remark this answers is not in the document]
Comment (ogabriel) [reply keyboards/durgod/k3x0/readme.md:55]: I also tested and really worked, i will update it

[the remark this answers is not in the document]
Comment (ogabriel) [reply keyboards/durgod/k3x0/readme.md]: done

File: keyboards/durgod/k3x0/readme.md (hunk 1 above)
Comment (ogabriel) [reply keyboards/durgod/k3x0/readme.md]: done

File: keyboards/durgod/k3x0/readme.md [hunk 3]
```diff
@@ -18,34 +18,58 @@ Instructions for building the K310 and K320 firmware can be found here:
...
 
-You can short Boot0 to Vdd by shorting R21 to C27 on the sides closest to the processor, as shown:
+1. Be in Bootloader mode.
+2. Here are a few options for performing the initial Flash:
+  - Using [QMK Toolbox](https://github.com/qmk/qmk_toolbox)
+  - Using qmk on command line:
 
-<img src="https://i.imgur.com/hvDnw5a.jpg" width="520" height="693" alt="Shorting Boot0 to Vdd on K320">
+```bash
+qmk flash
```
Comment (fauxpark) [inline keyboards/durgod/k3x0/readme.md:66]:
```suggestion
qmk flash -kb durgod/k3x0 -km default
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/durgod/k3x0/readme.md (hunk 3 above)
Comment (ogabriel) [reply keyboards/durgod/k3x0/readme.md]: using only `durgod/k3x0` will always build k310 file, so i put both commands for each variation.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**